server/logger.js

```diff
@@ -32,7 +31,7 @@ const stringify = format((info) => {
 });
 
 const logger = createLogger({
-  level: LOGGER_CONFIG.level,
+  level: process.env.LOGGER__LEVEL || 'silly',
```

Comment:
Why do you use `process.env` ? `LOGGER__LEVEL` is define in `.env` file, isn't it ?

Reply:
Yeah, I couldn't use `nconf` here because it is initialized just after startup (before nconf). So I must use process.env (provided by `dotenv`) instead of a `nconf.get('logger.level')`